Libplanet/Hashcash.cs

```diff
@@ -40,12 +40,12 @@ public static class Hashcash
         /// <param name="stamp">A callback to get a &#x0201c;stamp&#x0201d;
         /// which is a <see cref="byte"/> array determined from a given
         /// <see cref="Nonce"/> value.</param>
-        /// <param name="difficulty">The minimum required number of
-        /// leading zero bits that a returned answer needs to have.</param>
+        /// <param name="difficulty">A number to calculate the target number
+        /// for which the returned answer should be less than.</param>
```

Comment:
Isn't the `difficulty` parameter in itself the *target number*?  Didn't you mean “the number that a nonce has to satisfy,” not “the number to calculate the target number” (I didn't get what it means though)?

Reply:
`difficulty` parameter itself isn't the target number. The target number is `2^256 / difficulty`.